### ISSUE TITLE
Revise / remove Geouttag reference in plugins.md

### ISSUE THREAD

**Issue by Grammostola** [origo-map/origo#1270]
https://github.com/Eskilstuna-kommun/Geouttag is mentioned in plugins.md . Unfortunately it was last updated to work with release 2.2 of Origo, further it employs https://docs.safe.com/fme/html/FME_Server_Documentation/ReferenceManual/architecture.htm web services rather than the rest api which potentially makes it a bit less flexible. Third its interface-building needs an overhaul as parts are hard-coded that should in the optimal case not be.
 In short it can be tricky to get working well and it needs attention, something it will get from us (eskilstuna-kommun) when and if its development is prioritized. 
We can either remove it from plugins.md or we may mention what version of Origo it was last known to work with. 
(There's also Hallstahammar's solution which isn't at present a plugin)
